src/main/resources/com/eprosima/uxr/idl/templates/SerializationTestSource.stg

```diff
@@ -78,6 +78,8 @@ int test$ctx.lastStructure.name$()
 
 int main(void)
 {
+    time_t t;
+    srand((unsigned) time(&t));
```

Comment:
I have seen, that in both, Linux and Windows, the parameter of `time` function can be NULL, so the `t`variable is not needed.